cmd/record.go

```diff
@@ -167,7 +167,19 @@ func (r *Record) GetCmd() *cobra.Command {
 
 			passThrough := []models.Filters{}
 
-			err = r.GetRecordConfig(&path, &proxyPort, &appCmd, &appContainer, &networkName, &delay, &buildDelay, &ports, &passThrough, configPath)
+			err = r.GetRecordConfig(&models.RecordOptions{
+				Path:             path,
+				ProxyPort:        proxyPort,
+				AppCmd:           appCmd,
+				AppContainer:     appContainer,
+				AppNetwork:       networkName,
+				Delay:            delay,
+				BuildDelay:       buildDelay,
+				Ports:            ports,
+				Filters:          &filters,
+				EnableTele:       enableTele,
+				PassThroughHosts: passThrough,
+			}, configPath)
 			if err != nil {
 				if err == errFileNotFound {
```

Comment:
instead of declaring it twice better to assign to a variable and use it in both options.

Reply:
This is viable for RecordOptions, Implemented